docs/Configuration.md

```diff
@@ -993,6 +993,31 @@ function testRunner(
 
 An example of such function can be found in our default [jasmine2 test runner package](https://github.com/facebook/jest/blob/master/packages/jest-jasmine2/src/index.js).
 
+### `testSequencer` [string]
+
+Default: `@jest/test-sequencer`
+
+This option allows you to use a custom sequencer instead of Jest's default test runner.
+
+You need to export a sequencer class and set path in the config
+
+example:
+
+Sort test path alphabetically
+
+```js
+const Sequencer = require('@jest/test-sequencer').default;
+
+class CustomSequencer extends Sequencer {
+  sort(tests) {
```

Comment:
Is this a good way to describe the type?
```js
const Sequencer = require('@jest/test-sequencer').default;

class CustomSequencer extends Sequencer {
  sort(tests: Array<Test>) {
    /**
     *  Test = {
     *    context: Context;
     *    duration?: number;
     *    path: Config.Path;
     *  };
     */
    const copyTests = Array.from(tests);
    return copyTests.sort((testA, testB) => (testA.path > testB.path ? 1 : -1));
  }
}

module.exports = CustomSequencer;
```
Or just keep the type of `tests`, not describe `Test`.

Reply:
That's better. But we could also avoid changing code and just add this note on the following core block, with a link to type in the Jest source (e.g. it's not clear what Context is). Also I'd use a "string" for path.

Reply:
Maybe we can give a link to `types` ?
```js
const Sequencer = require('@jest/test-sequencer').default;

class CustomSequencer extends Sequencer {
  sort(tests: Array<Test>) {
    // Test structure information
    // https://github.com/facebook/jest/blob/master/packages/jest-runner/src/types.ts#L17
    const copyTests = Array.from(tests);
    return copyTests.sort((testA, testB) => (testA.path > testB.path ? 1 : -1));
  }
}

module.exports = CustomSequencer;
```
Or just keep the type of `tests`, not describe `Test`.

Reply:
we just need to make sure they're up-to-date, so we need to link to a commit or tag

Reply:
Sorry, I don't fully understand.
Could you give me some examples?

Reply:
E.g.: 

```
https://github.com/facebook/jest/blob/6b8b1404a1d9254e7d5d90a8934087a9c9899dab/packages/jest-runner/src/types.ts#L17-L21 
```

https://github.com/facebook/jest/blob/6b8b1404a1d9254e7d5d90a8934087a9c9899dab/packages/jest-runner/src/types.ts#L17-L21 

Now the link will always point to the valid type because it's connected with a certain commit in the repo. If you point to master – it may change. 

You can use "copy permalink" to get that easily:

![image](https://user-images.githubusercontent.com/5106466/55160262-3b4c9e80-5163-11e9-9dad-c10d1dc48dd6.png)

Reply:
Got it!
Thank you very much.